app/models/preneeds/applicant_input.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+module Preneeds
+  class ApplicantInput < Common::Base
+    include ActiveModel::Validations
+
+    validate :validate_name, if: -> (v) { v.name.present? }
```

Comment:
Name is complex and has `last_name`, `first_name`, `middle_name`, `maiden_name`, and `suffix` elements. only last_name and first_name are required.